src/inspector_socket_server.cc

```diff
@@ -25,9 +25,18 @@ void Escape(std::string* string) {
 }
 
 std::string GetWsUrl(const std::string& host, int port, const std::string& id) {
-  char buf[1024];
-  snprintf(buf, sizeof(buf), "%s:%d/%s", host.c_str(), port, id.c_str());
-  return buf;
+  // Host is valid (socket was bound) so colon means it's a v6 IP address
```

Comment:
Should we merge this with the code in `inspector_agent.cc@node::inspector::URL()` after the if?
/cc @sam-github